### PR TITLE
PE-1038-20220713: Weekly Executive (Görli)

### PR DESCRIPTION
**Pull request by hexonaut**
# Description

# Contribution Checklist

- [ ] PR title starts with `(PE-<TICKET_NUMBER>)`
- [ ] Code approved
- [ ] Tests approved
- [ ] CI Tests pass

# Checklist

- [ ] Every contract variable/method declared as public/external private/internal
- [ ] Consider if this PR needs the `officeHours` modifier override
- [ ] Verify expiration (`30 days` unless otherwise specified)
- [ ] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [ ] Validate all addresses used are in Goerli changelog or known
- [ ] Notify any external teams affected by the spell so they have the opportunity to review
- [ ] Deploy spell to Goerli `ETH_GAS="XXX" ETH_GAS_PRICE="YYY" make deploy`
- [ ] Ensure contract is verified on `Goerli` etherscan
- [ ] Change test to use Goerli spell address and deploy timestamp
- [ ] Cast spell on Goerli `make spell="0x-deployed-spell-address" cast-spell`
- [ ] Run `make archive-spell` or `make date="YYYY-MM-DD" archive-spell` to make an archive directory and copy `Goerli-DssSpell.sol`, `Goerli-DssSpell.t.sol`, `Goerli-DssSpell.t.base.sol`, and `Goerli-DssSpellCollateralOnboarding.sol`
- [ ] `squash and merge` this PR

### CODE REVIEW

File: src/Goerli-DssSpell.sol [hunk 1]
```diff
@@ -30,25 +30,65 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
...
+    }
 
     function actions() public override {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralOnboardingAction
         // onboardNewCollaterals();
-        offboardCollaterals();
+        // offboardCollaterals();
+
+        // Increase the GUNIV3DAIUSDC1-A Stability Fee from 0% to 0.01%.
```
Comment (naszam) [inline src/Goerli-DssSpell.sol:58]:
```suggestion
        // MOMC Parameter Changes
        // https://vote.makerdao.com/polling/QmefrhsE#poll-detail

        // Increase the GUNIV3DAIUSDC1-A Stability Fee from 0% to 0.01%.
```

**Review by naszam — COMMENTED** [review]
Checked:

- Diffed against PE-1038 Mainnet (actions and config match)
- Test PASS 2c6dc15
```rust
Running 1 test for src/test/starknet.t.sol:StarknetTests
[PASS] testStarknet() (gas: 942774)
Test result: ok. 1 passed; 0 failed; finished in 157.53ms

Running 14 tests for src/Goerli-DssSpell.t.sol:DssSpellTest
[PASS] testCastCost() (gas: 747960)
[PASS] testFailTooEarly() (gas: 12550)
[PASS] testFailTooLate() (gas: 12549)
[PASS] testFailWrongDay() (gas: 12572)
[PASS] testFail_notScheduled() (gas: 14139)
[PASS] testOnTime() (gas: 744371)
[PASS] testSpellIsCast_GENERAL() (gas: 22659457)
[PASS] test_auth() (gas: 14315853)
[PASS] test_auth_in_sources() (gas: 4516134)
[PASS] test_bytecode_matches() (gas: 1367249)
[PASS] test_chainlog_values() (gas: 5498033)
[PASS] test_chainlog_version_bump() (gas: 3024401)
[PASS] test_nextCastTime() (gas: 341486)
[PASS] test_use_eta() (gas: 340274)
Test result: ok. 14 passed; 0 failed; finished in 2.08s
```

**Review by naszam — APPROVED** [review]
LGTDeploy

**Review by talbaneth — COMMENTED** [review]
Changes look good and match mainnet.
Tests pass.
LGTM to deplloy.

**Comment by naszam** [comment]
just noticed, we should cleanup the previous offboarding action in `Goerli-DssSpellCollateral.sol`

**Comment by hexonaut** [comment]
Spell good for final review: https://goerli.etherscan.io/address/0xc4BCED0F4f141915dcCE58DbAB2D2DEe88af472B

**Comment by talbaneth** [comment]
@hexonaut can you add the address and timestamp to the config file?

**Review by talbaneth — APPROVED** [review]
Recent changes ok.
Tests pass with deployed spell.
Good to cast and archive.

Running 1 test for src/test/starknet.t.sol:StarknetTests
[PASS] testStarknet() (gas: 959674)
Test result: ok. 1 passed; 0 failed; finished in 162.05ms

Running 14 tests for src/DssSpell.t.sol:DssSpellTest
[PASS] testCastCost() (gas: 765093)
[PASS] testFailTooEarly() (gas: 12572)
[PASS] testFailTooLate() (gas: 12549)
[PASS] testFailWrongDay() (gas: 12594)
[PASS] testFail_notScheduled() (gas: 14139)
[PASS] testOnTime() (gas: 761504)
[PASS] testSpellIsCast_GENERAL() (gas: 22679002)
[PASS] test_auth() (gas: 9223336852494789973)
[PASS] test_auth_in_sources() (gas: 9223336852484725565)
[PASS] test_bytecode_matches() (gas: 1371096)
[PASS] test_chainlog_values() (gas: 5838809)
[PASS] test_chainlog_version_bump() (gas: 3195971)
[PASS] test_nextCastTime() (gas: 338878)
[PASS] test_use_eta() (gas: 337755)
Test result: ok. 14 passed; 0 failed; finished in 588.30ms

**Review by naszam — APPROVED** [review]
Checked:

- Deployed Spell Verified
    - Optimization Enabled: No
    - Other Settings: default evmVersion, GNU AGPLv3 license
- Deployed Spell Code Matches GitHub
    - [diffcheker](https://www.diffchecker.com/w1WW3rKK) agaist spell pr (ethescan code matches pr)
- Deploy Timestamp Match
- DssExecLib v0.0.9
- Test PASS (with deployed spell at ef1eda6):
```haskell
dapp-test: rpc block: latest
Running 14 tests for src/Goerli-DssSpell.t.sol:DssSpellTest
[PASS] testFail_notScheduled() (gas: 12139)
[PASS] test_auth_in_sources() (gas: 4449479)
[PASS] test_use_eta() (gas: 338874)
[PASS] testOnTime() (gas: 704216)
[PASS] test_nextCastTime() (gas: 339886)
[PASS] test_chainlog_values() (gas: 4863378)
[PASS] testFailTooEarly() (gas: 8550)
[PASS] test_bytecode_matches() (gas: 1365249)
[PASS] test_chainlog_version_bump() (gas: 2982246)
[PASS] test_auth() (gas: 14249198)
[PASS] testSpellIsCast_GENERAL() (gas: 21867818)
[PASS] testFailWrongDay() (gas: 8572)
[PASS] testCastCost() (gas: 705805)
[PASS] testFailTooLate() (gas: 8549)

Running 1 tests for src/test/starknet.t.sol:StarknetTests
[PASS] testStarknet() (gas: 880119)
```

**Review by talbaneth — APPROVED** [review]
Archive ok, good to merge.

**Comment by hexonaut** [comment]
Note: Archive was on the wrong day - updated it to yesterday.

**Review by naszam — APPROVED** [review]
LGTM

**Pull request merged**